crates/typst-library/src/model/outline.rs

```diff
@@ -59,16 +70,89 @@ use crate::text::{LinebreakElem, LocalName, SpaceElem, TextElem};
 /// ```
 ///
 /// # Styling the outline
-/// The outline element has several options for customization, such as its
-/// `title` and `indent` parameters. If desired, however, it is possible to have
-/// more control over the outline's look and style through the
-/// [`outline.entry`]($outline.entry) element.
-#[elem(scope, keywords = ["Table of Contents"], Show, ShowSet, LocalName)]
+/// At the most basic level, you can style the outline by setting properties on
+/// it and its entries. This way, you can customize the outline's
+/// [`title`]($outline.title), how outline entries are
```

Comment:
```suggestion
/// [title]($outline.title), how outline entries are
```

It's weird that only this link is displayed in a code font in the paragraph. In the context of the sentence, "title" refers to the concept, not the exact argument. Therefore, I would argue it should not be typeset in a code font.